index.html

```diff
@@ -31,4 +31,16 @@
 		<p>The download starts automatically when you visit pass the link to the GitHub directory as <code>url</code> parameter, like:</p>
 		<p><a href="https://download-directory.github.io/?url=https://github.com/mrdoob/three.js/tree/dev/build"><strong>download-directory.github.io</strong><code>?url=https://github.com/mrdoob/three.js/tree/dev/build</code></a></p>
 	</div>
+	<div class="error" id="error-repo-not-found">
+		<p>The requested repository was not found. It either does not exist or your provided API token does not grant access to it.</p>
+	</div>
+	<div class="error" id="error-network">
+		<p>You are not connected to the internet.</p>
+	</div>
+	<div class="error" id="error-adblocker">
+		<p>The downloading of a file may have been blocked by an ad blocker or similar software. Please disable blockers for this site and refresh this page to try again.</p>
+	</div>
+	<div class="error" id="error-private-repo">
+		<p>The requested directory is part of a private repository, downloading which is <a href="https://github.com/download-directory/download-directory.github.io/issues/7">not yet supported</a>.</p>
+	</div>
```

Comment:
I don't want to keep these error in the DOM, `updateStatus` supports HTML too

Reply:
As for the style, an emoji might be good enough: ⚠

Reply:
Wouldn't those error messages be quite... massive? I mean, the status is quite big and I tried to include some explanation for the user. I could just include some HTML that makes the explanation smaller — or would you reduce the error messages altogether?

Reply:
That's what I mean, with the explainer already compacted a bit.

![Massive error message](https://user-images.githubusercontent.com/1922624/67865399-7fec4e00-fb27-11e9-9142-9ccc9c2e3a2e.png)

Reply:
Ahhh I see what you mean. I hope to improve the style of the text at some point, but the first screenshot isn't too bad, and it's clear. ifttt.com has larger text 😅